Source/WebCore/dom/Document.h

```diff
@@ -998,6 +998,8 @@ class Document
     void updateViewportArguments();
     void processReferrerPolicy(const String& policy, ReferrerPolicySource);
 
+    void metaElementColorSchemeChanged();
```

Comment:
I think this should be behind `#if ENABLE(DARK_MODE_CSS)` like `processColorScheme`?